### DISCUSSION TITLE
Refactor code

### DISCUSSION THREAD

**Discussion by zrock-dev** [zrock-dev/Mr.Meeseeks#2] [Ideas]
What parts of the code do we have to change.

**Comment by zrock-dev** [comment]
Que en la consola no se muestre el numero que debemos adivinar

**Reply by EmanuelGalindoC** [reply]
Eliminar el codigo innecesario

**Reply by luiggy2620** [reply]
Cambiar en nombre las variables y metodos para que sea legible, usando camel case por ejemplo

**Reply by luiggy2620** [reply]
Separar todo lo que hace el main en metodos o clases

**Reply by luiggy2620** [reply]
cada vez que el usario termine, el numero de intentos en que lo hizo se reinicie

**Reply by JCarlosHidalgo** [reply]
Reestructurar el sistema de victoria en un comparador en lugar de un contador.

**Reply by EmanuelGalindoC** [reply]
Cambiar algunos printlns

**Reply by zrock-dev** [reply]
Crear una funcion que imprima el titulo al inicio del juego.

**Reply by JCarlosHidalgo** [reply]
Crear un sistema de recompensas que varíe de acuerdo al número de intentos.

**Reply by zrock-dev** [reply]
Crear una funcion para pedir entrada de datos al usuario y que nos retorne ese valor

**Reply by EmanuelGalindoC** [reply]
Aplicar los principios de responsabilidad

**Comment by zrock-dev** [comment]
Una funcion que compare los numeros y te diga si el numero es mayor o menor.
`                if (number < hiddernumber) {

                    System.out.println("***************************************");

                    System.out.println("Enter a Major Number to " + number);

                } else {

                    System.out.println("***************************************");

                    System.out.println("Enter a Less Number to " + number);

                }`

Que se ocupe de esta parte del codigo

**Reply by luiggy2620** [reply]
Separar todos los System.out.println en metodos diferentes, cosa que haya metodos para mostrar al usuario o otros metodos encargados  de la parte logica

**Comment by zrock-dev** [comment]
Un metodo para cuando gane el usuario.
`              if (number == hiddernumber) {

                System.out.println("***************************************");

                System.out.println("Congratulations! the Number hidden was " + hiddernumber);

                win++;

            }`

**Comment by zrock-dev** [comment]
Hay que sacar todo el codigo del main, y ponerlo en una clase guess number

**Comment by JCarlosHidalgo** [comment]
Otorgar una recompensa que mientras menor sea el número de intentos hasta llegar al número oculto más llamativa sea para alentar al usuario a seguir jugando.

**Comment by luiggy2620** [comment]
Separar los metodos en clases diferentes segun la capa de arquitectura

**Comment by JCarlosHidalgo** [comment]
cambiar la lógica del sistema de juego ganado a un comparador en lugar de un contador.

**Comment by luiggy2620** [comment]
Los metodos y variables tengan modificadores de acceso que no sean publicos si no es necesario
